### PR TITLE
Feature/david/remove bottom bar

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/72649045549333/1177859689086266
Tech Design URL: 
CC: 

**Description**:
The Bottom Navigation experiment is now concluded and we've agreed to remove it for now. This PR removes all the code from the experiment.

**Steps to test this PR**:
1. Open the app
2. Ensure ""mb" variant is no longer appearing
3. There should be no traces of bottom navigation
4. Open Tab Switcher screen
5. There should be no traces of bottom navigation

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt [hunk 1]
```diff
@@ -1282,68 +1264,51 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
...
 
-        fun decorateWithToolbar() {
-            hideBottomBar()
-            decorateAppBarWithToolbarOnly()
-            createPopupMenuWithToolbarOnly()
-            configureShowTabSwitcherListenerWithToolbarOnly()
-            configureLongClickOpensNewTabListenerWithToolbarOnly()
-            removeUnnecessaryLayoutBehaviour()
+        fun decorateToolbarMenus(viewState: BrowserViewState) {
+            decorator.updateToolbarActionsVisibility(viewState)
```
Comment (cmonfortep) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt:1276]: replace method call with `updateToolbarActionsVisibility` implementation? Also to simplify method forwarding for just one line.

File: app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherBottomBarFeatureActivity.kt [hunk 2]
```diff
@@ -92,7 +90,6 @@ class TabSwitcherBottomBarFeatureActivity : DuckDuckGoActivity(), TabSwitcherLis
         extractIntentExtras()
```
[the remark this answers is not in the document]
Comment (malmstein) [reply app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherBottomBarFeatureActivity.kt:90]: Yes, you are right! I"m removing all those files

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt [hunk 3]
```diff
@@ -732,15 +721,8 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
         autoCompleteSuggestionsList.adapter = autoCompleteSuggestionsAdapter
     }
 
-    private fun isBottomNavigationFeatureEnabled() =
-        variantManager.getVariant().hasFeature(VariantManager.VariantFeature.BottomBarNavigation)
-
     private fun decorateWithFeatures() {
```
[the remark this answers is not in the document]
Comment (malmstein) [reply app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt:724]: that's a good idea, no need to have calls to other methods if they are only called from one place

**Comment by malmstein** [comment]
@cmonfortep I've added the changes you requested. Thanks for the review!

**Pull request merged**